Source/dom/models/ExportFormat.js

```diff
@@ -0,0 +1,109 @@
+import { DefinedPropertiesKey, WrappedObject } from '../WrappedObject'
+import { Types } from '../enums'
+import { Factory } from '../Factory'
+
+/**
+ * An MSExportFormat. This is not exposed, only used by the exportOptions of the Layer
+ */
+export class ExportFormat extends WrappedObject {
+  constructor(exportFormat = {}) {
+    if (!exportFormat.sketchObject) {
+      // eslint-disable-next-line no-param-reassign
+      exportFormat.sketchObject = MSExportFormat.formatWithScale_name_fileFormat(
```

Comment:
Maybe we can use `[MSExportOptions nextMostAppropriateExportFormatToAdd:]` here, so as not to hardcode the values for the initial exportFormat?

Reply:
Hum I'm not sure. I get why we want that in the UI but IMO it would be weird to have a different fileformat depending on which one was added before